### DISCUSSION TITLE
MIDI implementation

### DISCUSSION THREAD

**Discussion by vertigocycles** [Tonfisch/TC-1128-Editor#3] [General]
Do you already have the CC list for the 1128?  If not, I’m happy to point you in the right direction. At some point in time I made a sysex chart for the front panel buttons that I’m happy to share as well

**Comment by Tonfisch** [comment]
Hello,
I know the cc's for eq, gain, volume and eq-on/off.
If there are more, i would be very interested.
The Key-Sysex are often in use in the editor. But i would like to see what you gatherd, may be there is info i don't have.
Many thanks for your input.
Greetings from Germany and sorry for the bad english
Toni
